languages/ruby/exercises/concept/numbers/.docs/after.md

```diff
@@ -0,0 +1,87 @@
+One of the key aspects of working with numbers in Ruby is the distinction between integers (numbers with no digits after the decimal separator) and floating-point numbers (numbers with zero or more digits after the decimal separator).
+They are implemented through the [`Integer`][integer-ruby] and [`Float`][float-ruby] class.
+
+```ruby
+a = 1
+b = 1.0
+a.class
+#=> Integer
+b.class
+#=> Float
+```
+
+- Arithmetic is done using the basic [arithmetic operators][arithmetic-operators] (`+`, `-`, `*`, `/`). Numbers can be compared using the standard [comparison operators][comparison-operators].
+- Basic arithmetic operations between instances of `Integer`, will always result in an instance of `Integer`.
+- Basic arithmetic operations between instances of `Float` will result in other instances of `Float`.
+- Basic arithmetic operations between instances of `Integer` and instances of `Float` will result in instances of `Float`.
+- The `Float` and `Integer` classes have methods that will coerce values from one to the other. `Integer` numbers are precise to a whole unit, while `Float` has precision that is fractional to an whole number. This means that coercing a float to an integer may result in loss of precision.
+
+```ruby
+4.9.to_i
+#=> 4
+
+5.to_f
+#=> 5.0
+
+5 * 6
+#=> 30
+
+1.2 > 0.8
+#=> true
+
+3.0 > 3
+#=> false
+
+7 - 3.0
+#=> 4.0
+
+2 != 4
+#=> true
```

Comment:
```suggestion
2 == 4
#=> false
```

Perhaps also add an example that does show equality check between Float and Integer:

```ruby
>> 1.0 == 1
=>t rue
```